### DISCUSSION TITLE
how to apply migrations from multiple folders

### DISCUSSION THREAD

**Discussion by tsvetann** [mikro-orm/mikro-orm#5083] [Q&A] [answered]
I have setup in which every team creates migration files in their own space (different folders). Is there an elegant way to tell Mikro ORM to apply all migrations across multiple folders? Thanks

**Comment by B4nan** [comment, marked as answer]
You could use `migrationsList` for this, doing the grouping yourself and just passing down an array of all found migrations to the ORM

https://mikro-orm.io/docs/migrations#importing-migrations-statically

You could also use `globby` to find all migrations in all folders based on a common name.
